framework/include/wifi_manager/wifi_manager.h

```diff
@@ -204,12 +226,24 @@ wifi_manager_result_e wifi_manager_get_info(wifi_manager_info_s *info);
 /**
  * @brief Connect to an access point.
  * @details @b #include <wifi_manager/wifi_manager.h>
- * @param[in] ssid, passphrase, authentication type, and cryto type of the access point which the wi-fi interface connect to.
+ * @param[in] config ssid, passphrase, authentication type, and cryto type of the access point which the wi-fi interface connect to.
+ * @param[in] conn_config
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
```

Comment:
Please, use TizenRT instead of Tizen RT.

Reply:
ok I will fix